openmdao/components/meta_model_structured_comp.py

```diff
@@ -8,9 +8,12 @@
 
 from scipy import __version__ as scipy_version
 try:
-    from scipy.interpolate._bsplines import make_interp_spline
+    from scipy.interpolate._bsplines import make_interp_spline as _make_interp_spline
 except ImportError:
-    make_interp_spline = False
+    def _make_interp_spline(*args, **kwargs):
```

Comment:
I wonder if you will ever keep separate efforts things to separate branches ??? :)